perun-core/src/main/java/cz/metacentrum/perun/core/bl/ResourcesManagerBl.java

```diff
@@ -360,6 +361,17 @@
 	 */
 	void assignGroupsToResource(PerunSession perunSession, Iterable<Group> groups, Resource resource, boolean async, boolean assignInactive, boolean autoAssignSubgroups) throws WrongAttributeValueException, WrongReferenceAttributeValueException, GroupResourceMismatchException;
 
+	/**
+	 * Asynchronously assigns single subgroup to resource as automatically assigned source group's subgroup.
+	 * Source group must have existing assignment on resource with autoAssignSubgroups set to true.
+	 * @param perunSession sess
+	 * @param sourceGroup source group (containing groupToAssign in hierarchy as a subgroup)
+	 * @param groupToAssign source group's subgroup to be assigned on resource as by automatic assignment
+	 * @param resource resource
+	 * @param assignInactive assign subgroup to inactive state
```

Comment:
Please, add here also the exceptions that can be thrown.

Reply:
Added.